scripts/_template_slurm_jobscript.batch

```diff
@@ -15,7 +15,8 @@
 
 
 ## LOAD MODULES, INSERT CODE, AND RUN YOUR PROGRAMS HERE
-module load libffi-3.3-gcc-9.3.0-cgokng6
+# module load libffi-3.3-gcc-9.3.0-cgokng6 : deprecated as of April-2025
+module load libffi-3.4.2-gcc-11.2.0-q7idpyn
```

Comment:
Would something like `module load libffi` work to allow easier longer term maintenance?

Reply:
I would say so, yes. Unless you believe this could be more problematic in the longer term.

Reply:
Agreed. I will make the change thanks Jorge